docs/agent/sensor-full-disk-access.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[sensor-full-disk-access]]
+== Enable full disk access
+
+Elastic cloud users can take advantage of Endgame's detection sensor. However, However, after installing the sensor on either macOS 10.14 Mojave or 10.15 Catalina, you must enable a setting called "Full Disk
+Access" to provide the sensor the ability to analyze files that are in your home directory.
+
+[float]
+=== Access to the Endgame sensor
+
+Review the https://www.elastic.co/support_policy/endpoint-system-requirements[Elastic Endpoint Security System Requirements]. Then contact Elastic support to gain access to the Endgame legacy sensor. This sensor is only available to current Elastic and Endgame customers.
```

Comment:
I think we should put a comma before "then" and make this once sentence. Let's change "Elastic Support" ==> Customer Success.

Reply:
Customer success is an internal name. Support feeds any of these requests to the Customer Success team unless customers are assigned an account manager.